README.md

```diff
@@ -122,6 +122,20 @@ user_dto = user.to_dto
 
 You can then use `user_dto` across your app, and even share it with other components, without having to worry about others making queries, modifying data, or even running business logic, where they shouldn't.
 
+This method accepts the same options as `ActiveRecord`'s `#as_json`, they are `except`, `only`, `methods`, and `include`.
+
+`except`
+Excludes attributes from the model when creating the DTO.
+
+`only`
+Selects which attributes should only be included when creating the DTO.
```

Comment:
Maybe something like this sounds a bit better?

Selects the only attributes that should be included when creating the DTO.